inc/user.class.php

```diff
@@ -4513,6 +4534,92 @@ static function getIdByField($field, $value) {
    }
 
 
+   /**
+    * Show password update form for current user.
+    *
+    * @param array $error_messages
+    *
+    * @return void
+    */
+   public function showPasswordUpdateForm(array $error_messages = []) {
```

Comment:
Original password fields are still present; I guess you should either remove them from original location, or factorize.

Reply:
As discussed, forms are differents; t is ok to keep as is